### ISSUE TITLE
Wrong assumptions of npm binary to install dependencies

### ISSUE THREAD

**Issue by nashfive** [xolvio/chimp#149]
```bash
$ brew install node --without-npm
$ echo prefix=~/.node >> ~/.npmrc
$ curl -L https://www.npmjs.com/install.sh | sh
$ echo 'export PATH="$HOME/.node/bin:$PATH"' >> ~/.bash_profile
```
Both ```node``` and ```npm``` are in the path :
```bash
$ which node && node -v
/usr/local/bin/node
v5.0.0
$ which node && node -v
/Users/xxxxx/.node/bin/npm
3.3.12
```
Running ```chimp``` for the first time should install selenium, etc :
```bash
$ chimp --watch
[chimp] stderr: /bin/sh: /usr/local/Cellar/node/5.0.0/bin/npm: No such file or directory

npm install error { [Error: Command failed: /bin/sh -c "/usr/local/Cellar/node/5.0.0/bin/npm" install
/bin/sh: /usr/local/Cellar/node/5.0.0/bin/npm: No such file or directory
]
  killed: false,
  code: 127,
  signal: null,
  cmd: '/bin/sh -c "/usr/local/Cellar/node/5.0.0/bin/npm" install' }
Error in chimp init { [Error: Command failed: /bin/sh -c "/usr/local/Cellar/node/5.0.0/bin/npm" install
/bin/sh: /usr/local/Cellar/node/5.0.0/bin/npm: No such file or directory
]
  killed: false,
  code: 127,
  signal: null,
  cmd: '/bin/sh -c "/usr/local/Cellar/node/5.0.0/bin/npm" install' }
```

It appears that chimp assumes that npm should be find in a subfolder of node installation. It's not always the case. 

See chimp.js at line 81: 
```javascript
var npmPath = path.join(path.dirname(process.execPath), 'npm');
```

**Comment by samhatoum** [comment]
As a workaround, you could create a symlink in there.

I'll see about a better way of determining where the npm binary is

**Comment by nashfive** [comment]
Thanks for your answer, that is what I did and it worked...
I'd suggest to make no assumptions about how and where NPM has been installed and just invoke it from the PATH, unless you need it to work in a very specific directory ?

**Comment by samhatoum** [comment]
Yes we should do that for Chimp in Node mode, but when running Meteor, we have to use the Meteor specific one. That's what the assumption is based on. It's not hard to do it :)

**Comment by nashfive** [comment]
Oh ok, sorry I haven't tried Chimp with Meteor...

**Comment by samhatoum** [comment]
No problem. I'll fix it soon!

**Comment by zvictor** [comment]
I am having the same issue here. However, I installed node with npm, default installation.

**Comment by zvictor** [comment]
+1

**Comment by zvictor** [comment]
I am linking to the npm from `/usr/local/bin` and it works.

`$ ln -s /usr/local/bin/npm npm`

**Comment by samhatoum** [comment]
Fixes in 0.24.0

**Comment by Davodu** [comment]
simple solution, download the traditional way https://nodejs.org/en/ then install(even if node is already installed), next npm -v, solved:)
